Test commit

cgbilke opened the pull request:
No commit changes

cgbilke committed:
No changes

Otokomidori reviewed (changes requested):
Files stopped working

cgbilke commented:
Not sure what the file bug was, should be corrected by updating UE4 from 4.16.2 to 4.16.3

cgbilke committed:
Map Update

Edited objects and checked into blueprints of character class to edit
some known bugs and blueprint logic.
